plasmapy/utils/exceptions.py

```diff
@@ -6,13 +6,13 @@
 """
 
 class PhysicsError(Exception):
-    """Error for using a deeply troubling physics value"""
+    """Error for using a worrisome physics value"""
     pass
 
 class PhysicsWarning(Warning):
     """Warning for using a mildly worrisome physics value"""
     pass
 
-class RelativityWarning(Warning):
+class RelativityWarning(PhysicsWarning):
     """Warning for approaching or exceeding the speed of light"""
     pass
```

Comment:
fancy inheritance! Is the idea that PhysicsWarning is the general class and more specific warnings will inherit from it? Do you forsee any cases where PhysicsWarning itself would be used outside of inheritance? If so, then you might need a more precise description of the use cases in the dosctring for PhysicsWarning.